Improved info about Picture in Picture support

Greg-21 opened the pull request:
**In this commit I did the following:**
* corrected and improved information about Picture in Picture support in web browsers
* changed links in "Links" section.

**Sources:**
* [Picture in Picture in Firefox Nightly](https://www.bleepingcomputer.com/news/software/mozilla-adding-a-picture-in-picture-mode-to-firefox/)
* [Picture in Picture in Chrome 69](https://techdows.com/2018/09/chrome-picture-in-picture-mode-vs-opera-video-pop-out.html)
* [Video Pop Out in Opera 37](https://blogs.opera.com/desktop/2016/05/ad-blocker-opera-for-windows-mac-free)
* [iOS 9 multitasking features](https://wccftech.com/ios-9-multitasking-features)
* [Picture in Picture on macOS 10.12 Sierra](https://support.apple.com/en-us/HT206997)

Schweinepriester commented:
Why remove the two links
- <https://developers.google.com/web/updates/2018/10/watch-video-using-picture-in-picture>
- <https://googlechrome.github.io/samples/picture-in-picture/>

?

Greg-21 commented:
I removed the first link mentioned by you, because it almost completely focuses on Chrome, which is not as much helpful as it could be. Sample source codes can be found on the Internet by everyone interested in :) Talking about the 2nd link, now I don't know why I removed it and I think it could be helpful. So I'm going to bring it back.

Fyrd commented:
Thanks!
